golden_response/staging/get_stat_all/result.json

```diff
@@ -1,698 +1,698 @@
 {
- "placeData": {
-  "country/USA": {
-   "statVarData": {
-    "Amount_EconomicActivity_GrossNationalIncome_PurchasingPowerParity_PerCapita": {
-     "sourceSeries": [
+ "placeData":  {
```

Comment:
@tjann I also got this formatting result from the command, can you try this locally? I don't think this should be related to vscode